Template10 (Library)/Controls/HamburgerMenu.xaml

```diff
@@ -168,9 +187,10 @@
                                             <Rectangle x:Name="CheckedBack"
                                                        Fill="{Binding NavButtonCheckedBackground, ElementName=ThisPage}"
                                                        IsHitTestVisible="False" Visibility="Collapsed" />
-                                            <ContentControl x:Name="NormalFore" AutomationProperties.AccessibilityView="Raw"
+                                            <ContentControl x:Name="NormalFore"
                                                             Content="{TemplateBinding Content}"
                                                             Foreground="{Binding NavButtonForeground, ElementName=ThisPage, FallbackValue=White}"
+                                                            IsTabStop="False"
                                                             IsHitTestVisible="False" />
```

Comment:
By default it will end up as a tab stop, so as you tab around it'll move down the xaml tree to each level of the nav button, radiobutton -> togglebutton -> etc.

To the user, the nav button is a single thing for them to click and interact with, so there should be a single tabstop for them to stop on and interact with?

The complexity of this Nav Button xaml hierarchy I think is a bit extreme with the different behaviors of the various nested control types interacting.